### ISSUE TITLE
New rule: avoid duplicate fields in operations

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [dotansimha/graphql-eslint#71]
It should suggest remove duplicate fields, input fields, arguments, etc.

As an option, it can suggest to use GraphQL aliases for duplicate fields instead of removing them.

**Comment by dotansimha** [comment]
@FluorescentHallucinogen Not sure I understood. Example for incorrect code? 
Thanks!

**Comment by FluorescentHallucinogen** [comment]
```graphql
query getUsers {
  users {
    id
    firstName
    lastName
    username
    phone
    email
    github
    email #twice
    twitter
  }
}
```

This already implemented in [graphql-prettier](https://www.npmjs.com/package/graphql-prettier).

**Comment by ilyavolodin** [comment]
https://github.com/dotansimha/graphql-eslint/blob/master/docs/rules/prettier.md

**Comment by FluorescentHallucinogen** [comment]
```graphql
query getUsers {
  users(
    first: 100
    skip: 50
    after: "cji629tngfgou0b73kt7vi5jo"
    first: 100 # duplicate argument
  ) {
  id
  }
}
```

**Comment by dotansimha** [comment]
Implemented in: https://github.com/dotansimha/graphql-eslint/pull/245

**Comment by dotansimha** [comment]
Available in `@graphql-eslint/eslint-plugin@0.7.0`
